thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/context/AnomalyDetectionContext.java

```diff
@@ -44,29 +46,29 @@ public void setTimeSeriesKey(TimeSeriesKey key) {
   /**
    * Returns the current (observed) time series, which is not transformed.
    */
-  public TimeSeries getCurrent() {
-    return current;
+  public TimeSeries getCurrent(String metricName) {
```

Comment:
related to previous comments in AnomalyContext. you need to use metricName anyway, so under this new situation, its better to treat current and baseline equally. Since both are modeled as timeseries right now. It should be fine to have 'un-equal' length for baseline and current. In general, as long as we remove the seasonality, you do not need the baseline and current always aligned in length. Usually the transformer do not mix current and baseline.